index.html

```diff
@@ -0,0 +1,779 @@
+        <li class="footer-nav__link">
+          <a href="#">Tools & Services</a>
+        </li>
+        <li class="footer-nav__link">
+          <a href="#">N.Y.C. Events Guide</a>
+        </li>
+        <li class="footer-nav__link">
+          <a href="#">Multimedia</a>
+        </li>
+        <li class="footer-nav__link">
+          <a href="#">Photography</a>
+        </li>
+        <li class="footer-nav__link">
+          <a href="#">Video</a>
+        </li>
+        <li class="footer-nav__link">
+          <a href="#">Newsletters</a>
+        </li>
+        <li class="footer-nav__link">
+          <a href="#">NYT Store</a>
+        </li>
+        <li class="footer-nav__link">
+          <a href="#">Times Journeys</a>
+        </li>
+        <li class="footer-nav__link">
+          <a href="#">Manage My Account</a>
+        </li>
+      </ul>
+    </nav>
+  </div>
+  <div class="footer-nav">
+    <h3 class="footer-nav__heading ">SUBSCRIBE</h3>
+    <nav>
+      <ul class="footer-nav__links">
+        <li class="footer-nav__link">
+          <a href="#">Home Delivery</a>
+        </li>
+        <li class="footer-nav__link">
+          <a href="#">Digital Subscriptions</a>
+        </li>
+        <li class="footer-nav__link">
+          <a href="#">Crossword</a>
+        </li>
+        <li class="footer-nav__link">
+          <a href="#">Cooking</a>
+        </li>
+        <li class="footer-nav__link">
+          <a href="#">Email Newsletters</a>
+        </li>
+        <li class="footer-nav__link">
+          <a href="#">Corporate Subscriptions</a>
+        </li>
+        <li class="footer-nav__link">
+          <a href="#">Education Rate</a>
+        </li>
+        <li class="footer-nav__link">
+          <a href="">Mobile Applications</a>
+        </li>
+        <li class="footer-nav__link">
+          <a href="#">Replica Edition</a>
+        </li>
+        <li class="footer-nav__link">
+          <a href="#">Español</a>
+        </li>
+      </ul>
+    </nav>
+  </div>
+  <div>
+    <nav id="conpyright-section">
+      <ul id="footer-bottom-nav">
+        <li><a id="footer-bottom-nav-a" href="#">©&nbsp;<span>2019</span><span>The New York Times Company</span></a></li>
+        <li><a id="footer-bottom-nav-a" href="#">Contact Us</a></li>
+        <li><a id="footer-bottom-nav-a" href="#">Work with us</a></li>
+        <li><a id="footer-bottom-nav-a" href="#">Advertise</a></li>
+        <li><a id="footer-bottom-nav-a" href="#">T Brand Studio</a></li>
+        <li><a id="footer-bottom-nav-a" href="#">Your Ad Choices</a></li>
+        <li><a id="footer-bottom-nav-a" href="#">Privacy</a></li>
+        <li><a id="footer-bottom-nav-a" href="#">Terms of Service</a></li>
+        <li><a id="footer-bottom-nav-a" href="#">Terms of Sale</a></li>
+        <li><a id="footer-bottom-nav-a" href="#">Site Map</a></li>
+        <li><a id="footer-bottom-nav-a" href="#">Help</a></li>
+        <li><a id="footer-bottom-nav-a" href="#">Subscriptions</a></li>
```

Comment:
There should be no ID selectors for multiple elements, ID should be unique for each element, better use class instead.